hacking/pr_labeler/pr_labeler/data/docs_team_info.md

```diff
@@ -1,2 +1,2 @@
-Thanks for your Ansible docs contribution! We talk about Ansible documentation on matrix at [#docs:ansible.im](https://matrix.to/#/#docs:ansible.im) and on libera IRC at #ansible-docs if you ever want to join us and chat about the docs! We meet there on Tuesdays (see [the Ansible calendar](https://github.com/ansible/community/blob/main/meetings/README.md)) and welcome additions to our [weekly agenda items](https://forum.ansible.com/t/documentation-working-group-agenda/153) - scroll down to find the upcoming agenda and add a comment to put something new on that agenda.
+Thanks for your Ansible docs contribution! We talk about Ansible documentation on matrix at [#docs:ansible.im](https://matrix.to/#/#docs:ansible.im) if you ever want to join us and chat about the docs! We meet on matrix every Tuesday. See [the Ansible calendar](https://forum.ansible.com/upcoming-events) for meeting details. We welcome additions to our [weekly agenda items](https://forum.ansible.com/t/documentation-working-group-agenda/153/7) too. You can add the `dawgs-meeting` tag to a forum topic to bring it up at the next meeting.
```

Comment:
The correct spelling is Matrix with an upper-case M BTW :) It's only the logo that uses a lower-case m.
```suggestion
Thanks for your Ansible docs contribution! We talk about Ansible documentation on Matrix at [#docs:ansible.im](https://matrix.to/#/#docs:ansible.im) if you ever want to join us and chat about the docs! We meet on Matrix every Tuesday. See [the Ansible calendar](https://forum.ansible.com/upcoming-events) for meeting details. We welcome additions to our [weekly agenda items](https://forum.ansible.com/t/documentation-working-group-agenda/153/7) too. You can add the `dawgs-meeting` tag to a forum topic to bring it up at the next meeting.
```